modules/apps/portal-workflow/portal-workflow-kaleo-service/src/main/java/com/liferay/portal/workflow/kaleo/service/impl/KaleoDefinitionServiceImpl.java

```diff
@@ -76,9 +143,32 @@ private void _checkPermissions(ServiceContext serviceContext)
 			ActionKeys.ADD_DEFINITION);
 	}
 
+	private void _checkViewPermissions(KaleoDefinition kaleoDefinition)
+		throws PortalException {
+
+		PermissionChecker permissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		if ((permissionChecker == null) ||
```

Comment:
Hey @pedro-oliveira446 do we still need this?

I thought could remove this after creating the thread local